src/content/docs/cachyos_basic/download.mdx

```diff
@@ -65,6 +65,29 @@ Always take an extra step and verify the ISOs integrity to avoid any undesired i
 
 </TabItem>
 
+<TabItem label='macOS'>
+
+<Steps>
+
+1. [Download](<https://mirror.cachyos.org/ISO/desktop/241221/cachyos-desktop-linux-241221.iso.sha256>) this file containing the SHA256 hash.
```

Comment:
```suggestion
1. Download the [file](https://mirror.cachyos.org/ISO/desktop/241221/cachyos-desktop-linux-241221.iso.sha256) containing the SHA256 hash.
```

Could you keep it consistent with #86? I'm letting sit for awhile in case there's anything I missed but it's going to be merged soon so I'd like to keep it consistent.

Reply:
Hope it's fine now?